<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>query</class>
 <widget class="QDialog" name="query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询图书</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>24</y>
     <width>661</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,3,1">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>请选择查询项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按ISBN号查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按书名查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按作者查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按出版社查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>图书一览</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>请输入查询内容</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
